sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegen.scala

```diff
@@ -284,7 +284,14 @@ case class WholeStageCodegen(plan: CodegenSupport, children: Seq[SparkPlan])
         ${ctx.declareAddedFunctions()}
 
         protected void processNext() throws java.io.IOException {
-          ${code.trim}
+          try {
+            ${code.trim}
+          } catch (final Throwable e) {
+            org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(this.getClass());
+            logger.error("The method processNext() is generated for " +
+                         "${plan.origin.callSite.getOrElse("unknown")}");
+            throw e;
+          }
```

Comment:
Hmm... Are there any case when generated expressions throw checked-exception?
In addition, Spark uses `Logging` class for logging so can we use it somehow?